### PR TITLE
Display balance in the DEX contract's open orders

### PR DESCRIPTION
**Pull request by jameswhollister**
## Ticket
* a652aacff4c903af6c44ab3991d8fc0f4d4ff412

## Changes
* Display balance in the DEX contract's open orders

### CODE REVIEW

File: src/renderer/services/dex.js [hunk 1]
```diff
@@ -278,7 +278,26 @@ export default {
         const currentWallet = wallets.getCurrentWallet();
         axios.get(`${currentNetwork.aph}/orders/${currentWallet.address}?contractScriptHash=${assets.DEX_SCRIPT_HASH}`)
           .then((res) => {
-            resolve(res.data.orders);
+            const orders = res.data.orders;
+            orders.forEach((o) => {
+              console.log(store.state.markets);
```
Comment (chovy) [inline src/renderer/services/dex.js]: remove console.log

File: src/renderer/services/dex.js [hunk 2]
```diff
@@ -317,6 +336,21 @@ export default {
     });
   },
 
+  fetchOpenOrderBalance(assetId) {
+    return new Promise((resolve, reject) => {
+      try {
+        console.log(store.state.orderHistory);
```
Comment (chovy) [inline src/renderer/services/dex.js]: remove console.log

File: src/renderer/services/dex.js [hunk 3]
```diff
@@ -278,7 +278,26 @@ export default {
         const currentWallet = wallets.getCurrentWallet();
         axios.get(`${currentNetwork.aph}/orders/${currentWallet.address}?contractScriptHash=${assets.DEX_SCRIPT_HASH}`)
           .then((res) => {
-            resolve(res.data.orders);
+            const orders = res.data.orders;
+            orders.forEach((o) => {
```
Comment (crishellco) [inline src/renderer/services/dex.js]: Single letter variables...

File: src/renderer/services/dex.js [hunk 4]
```diff
@@ -317,6 +336,21 @@ export default {
...
+    return new Promise((resolve, reject) => {
+      try {
+        console.log(store.state.orderHistory);
+        const openOrdersForAsset = _.filter(store.state.orderHistory, (order) => {
+          return order.assetIdToGive === assetId && (order.status === 'Open' || order.status === 'PartiallyFilled');
+        });
+
+        resolve(_.sumBy(openOrdersForAsset, 'quantityToGive'));
+      } catch (e) {
+        reject(e);
```
Comment (crishellco) [inline src/renderer/services/dex.js]: Let's reject explicit errors.

**Pull request merged**